kustomize/base/prometheus/prometheus.yml

```diff
@@ -131,4 +131,4 @@ scrape_configs:
   - action: replace
     source_labels:
     - __meta_kubernetes_pod_name
-    target_label: kubernetes_pod_name
+    target_label: kubernetes_pod_name
```

Comment:
Please undo this change. Also please add a new line at the end of files where that's missing (GH shows you where that's needed).